### PR TITLE
New documentation

### PR DESCRIPTION
**Pull request by pedrocamargo**

### CODE REVIEW

File: aequilibrae/parameters.py [hunk 1]
```diff
@@ -54,12 +53,38 @@ def __init__(self, project=None):
...
+        """Restores parameters to generic default
+        
+        .. code-block:: python
+        
+            from aequilibrae import Parameters
+
+            fldr = 'D:/myProject/logs'
+
+            p = Parameters()
+            p.reset_default()
```
Comment (pedrocamargo) [inline aequilibrae/parameters.py:85]:
```suggestion
            p.restore_default()
```

File: aequilibrae/parameters.py [hunk 2]
```diff
@@ -6,8 +6,7 @@
 
 
 class Parameters:
-    """
-    Global parameters module
+    """Global parameters module
```
Comment (pedrocamargo) [inline aequilibrae/parameters.py:9]: The parameters are saved to temp AND to the model folder

File: aequilibrae/parameters.py [hunk 3]
```diff
@@ -54,12 +53,38 @@ def __init__(self, project=None):
             self.parameters = deepcopy(self._default)
 
     def write_back(self):
-        """Writes the parameters back to file"""
+        """Writes the parameters back to file
```
Comment (pedrocamargo) [inline aequilibrae/parameters.py:56]: If we make sure that this is code that can be run, then we can actually test the docstrings.

What we can do is, in the beginning of the tests (before pytest), we create an example in the tempfolder (gettempdir()/"sfalls_just_to_open") and we can use that example everywhere.

File: docs/source/development/softwaredevelopment.rst [hunk 4]
```diff
@@ -123,9 +122,9 @@ In a more verbose way...
     * Unit testing
     * Build/packaging tests
     * Documentation building test
-* If the tests pass, then a manual pull request can be approved to merge into master.
-* The master branch is protected and therefore can only be written to after the code has been reviewed and approved.
-* No individual has the privileges to push to the master branch.
+* If the tests pass, then a manual pull request can be approved to merge into develop.
+* The master ande velop branches are protected and therefore can only be written to after the code has been reviewed and approved.
```
Comment (jamiecook) [inline docs/source/development/softwaredevelopment.rst:126]:
```suggestion
* The master ande velop branches are protected and therefore can only be written to after the code has been reviewed and approved.
```
```suggestion
* The master and develop branches are protected and therefore can only be written to after the code has been reviewed and approved.
```

File: docs/source/development/softwaredevelopment.rst [hunk 5]
```diff
@@ -155,13 +154,10 @@ Testing
...
-* `pytest <http://pytest.org/latest/>`_, a Python testing tool
-* `coveralls <https://github.com/coagulant/coveralls-python>`_, a tool for measuring test code coverage
+* `Black <https://black.readthedocs.io/en/stable/index.html/>`_, The uncompromising code formatter
 
-To run the tests locally, you will need to figure out what to do...
-
-
-These same tests are run by Travis with each push to the repository.  These tests need to pass in order to somebody
+Testing is done for Windows, MacOs and Ubuntu Linux on all supported Python versions, and we use GitHub Actions
+to run these tests. These tests need to pass in order to somebody
```
Comment (jamiecook) [inline docs/source/development/softwaredevelopment.rst:160]:
```suggestion
to run these tests. These tests need to pass and additionally somebody has to
```

**Review by jamiecook — APPROVED** [review]

**Pull request merged**